dashboards/eks/eks_cluster_dashboard.sp

```diff
@@ -0,0 +1,457 @@
+dashboard "aws_eks_cluster_dashboard" {
+
+  title         = "AWS EKS Cluster Dashboard"
+  documentation = file("./dashboards/eks/docs/eks_cluster_dashboard.md")
+
+  tags = merge(local.eks_common_tags, {
+    type = "Dashboard"
+  })
+
+  #cards
+
+  container {
+
+    # Analysis
+    card {
+      sql   = query.aws_eks_cluster_count.sql
+      width = 2
+    }
+
+    #Assessments
+    card {
+      sql   = query.aws_eks_cluster_secrets_encryption_disabled.sql
+      width = 2
+    }
+
+    card {
+      sql   = query.aws_eks_cluster_endpoint_endpoint_public_access_disabled.sql
+      width = 2
+    }
+
+    card {
+      sql   = query.aws_eks_cluster_audit_logging_disabled.sql
+      width = 2
+    }
+
+  }
+
+  # Assessments
+
+  container {
+
+    title = "Assessments"
+    width = 6
+
+    chart {
+      title = "Secrets Encryption Status"
+      sql   = query.aws_eks_cluster_secrets_encryption_status.sql
+      type  = "donut"
+      width = 4
+
+      series "count" {
+        point "enabled" {
+          color = "ok"
+        }
+        point "disabled" {
+          color = "alert"
+        }
+      }
+    }
+
+    chart {
+      title = "Endpoint Public Access Status"
+      sql   = query.aws_eks_cluster_endpoint_public_access_status.sql
+      type  = "donut"
+      width = 4
+
+      series "count" {
+        point "enabled" {
+          color = "ok"
+        }
+        point "disabled" {
+          color = "alert"
+        }
+      }
+    }
+
+    chart {
+      title = "Cluster Audit Logging Status"
+      sql   = query.aws_eks_cluster_audit_logging_status.sql
+      type  = "donut"
+      width = 4
+
+      series "count" {
+        point "enabled" {
+          color = "ok"
+        }
+        point "disabled" {
+          color = "alert"
+        }
+      }
+    }
+  }
+
+  container {
+
+    title = "Cost"
+    width = 6
+
+    table {
+      width = 6
+      title = "Forecast"
+      sql   = query.aws_eks_monthly_forecast_table.sql
+    }
+
+    chart {
+      width = 6
+      type  = "column"
+      title = "Monthly Cost - 12 Months"
+      sql   = query.aws_eks_cost_per_month.sql
+    }
+
+  }
+
+  # Analysis
```

Comment:
6 charts - 4 width (so it’s 3 then 3 per row)
Refer [here](https://github.com/turbot/steampipe-utils/wiki/Steampipe-Report-Development-Guide-(WIP)#dashboards)